packages/plugin-persistent-cache/src/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { createHash } from 'node:crypto'
+import fs from 'node:fs'
+import path from 'node:path'
+
+export function getCodeHash(code: string): string {
```

Comment:
this could probably just be called `getHash`. it will work regardless of whether the input is a "code" or not